### PR TITLE
Add a BSD license to the source code

### PR DESCRIPTION
**Pull request by RaymondMichael**
Adding a BSD license to the source code to clarify the permissions for others to copy bits of the standard into their own documentation.

### CODE REVIEW

**Comment by swpoole** [comment]
Looks good. I have also refreshed:
http://www.openshmem.org/site/node/58

**Comment by shamisp** [comment]
@jdinan Is there diff version of the document that can be shared with "legal" ?

**Comment by jdinan** [comment]
If you're looking for a diff:
https://github.com/openshmem-org/specification/pull/418

I attached the PDF for this draft.

 ~Jim.

**Comment by jdinan** [comment]
Superseded for OpenSHMEM 1.5 by #428. Pushing to OpenSHMEM 1.6 along with #313.

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Copyright 2011-2020 Hewlett Packard Enterprise Development LP
```
Comment (jdinan) [inline LICENSE.txt:1]:
```suggestion
Copyright 2011-2024 Hewlett Packard Enterprise Development LP
Copyright 2011-2024 Open Source Software Solutions
```

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Copyright 2011-2024 Hewlett Packard Enterprise Development LP
+Copyright 2011-2024 Open Source Software Solutions, Inc.
```
Comment (jdinan) [inline LICENSE.txt:2]:
```suggestion
Copyright 2011-2024 Open Source Software Solutions, Inc.
```

Comment (jdinan) [reply LICENSE.txt:2]: @swpoole Please review whether we can make this change.

**Pull request closed without merging**